CHANGES.rst

```diff
@@ -28,13 +28,17 @@ Improvements
   by users submitting an abstract (:issue:`3138`)
 - Add support for boolean (yes/no) and freetext questions in abstract
   reviewing (:issue:`3175`)
+- Add session types to sessions; allow marking session types as poster
+  so that the sessions with poster session type are displayed differently
+  in the timetable
 
 Internal Changes
 ^^^^^^^^^^^^^^^^
 
-- Rename *Roles* in ACL entries to *Permissions*.  This especially affects
+- Rename *Roles* in ACL entries to *Permissions*. This especially affects
   the ``can_manage`` method whose ``role`` argument has been renamed to
   ``permission`` (:issue:`3057`)
+- Change poster sessions to sessions with poster session type
```

Comment:
I don't think this is really worth mentioning here; the idea of the "internal changes" section is to point out things likely to be relevant to developers